tests/adapters/structures/utils.py

```diff
@@ -8,7 +8,7 @@ def get_min_ver(dependency: str) -> str:
     with open(setup_py, "r") as setup_file:
         for line in setup_file.readlines():
             min_ver = re.findall(
-                fr'"{dependency}((=|!|<|>|~)=|>|<)([0-9]+(\.[0-9]+){{,2}})"', line
+                fr'"{dependency}((=|!|<|>|~)=|>|<)([0-9]+(\.[0-9]+){{,3}})"', line
```

Comment:
I think we can get away with this, rather than making it even more general...

Reply:
Maybe it's better to indeed make it more general?
```suggestion
                fr'"{dependency}((=|!|<|>|~)=|>|<)([0-9]+(\.[0-9]+)+)"', line
```

Reply:
Looks like `importorskip` is using [Packaging](https://packaging.pypa.io/en/latest/version/) under the hood, which seems to crazy variants of versioning, so really I guess we should take whatever string follows the package?

Reply:
Yeah, you're probably right. So something like
```suggestion
                fr'"{dependency}((=|!|<|>|~)=|>|<)(.+)"', line
```

Reply:
Yep, that's exactly what I went for :)